target/linux/ar71xx/image/generic.mk

```diff
@@ -1,3 +1,20 @@
+define Device/OpenMesh
+  FILESYSTEMS := squashfs
+  DEVICE_PACKAGES := om-watchdog
+  KERNEL := kernel-bin | lzma | uImage lzma
+  IMAGES := sysupgrade.bin factory.bin
+  IMAGE/sysupgrade.bin/squashfs := append-rootfs | pad-rootfs | combined-ext-image
```

Comment:
As written, in my build test using the default config neither the ```FILESYSTEMS := squashfs``` nor ```/squashfs``` was required. But since ```FILESYSTEMS := squashfs``` isn't set in ```target/linux/ar71xx/Makefile.mk``` as I would expected it, I'm fine to keep this line.